src/resources/i18n/exchange/Messages_fr_FR.xml

```diff
@@ -1163,8 +1163,8 @@ L'antispam Vade Secure est compatible avec le mode non autoritatif (uniquement s
    <translation id="exchange_ACTION_configure_services_label_minPasswordLength" qtlid="174828">Longueur minimale du mot de passe</translation>
    <translation id="exchange_ACTION_configure_services_label_complexityEnabled" qtlid="178003">Exigences de complexité</translation>
    <translation id="exchange_ACTION_configure_services_label_incoming_messages" qtlid="81871">Messages entrants</translation>
-   <translation id="exchange_ACTION_configure_services_label_spf" qtlid="402399">Activer la protection SPF</translation>
```

Comment:
I think you can keep the qtlid with theses changes. To confirm :)

Reply:
I don't think so, the meaning of the sentence changed